app/views/public_submissions/add_author.html.haml

```diff
@@ -0,0 +1,25 @@
+.scroll-panel-header
+  %h3.scroll-panel-title Dodaj autora
+.scroll-panel-body
+  %h4 Autorzy
+  %table.table.table-hover.table-admin
+    %thead
+      %tr
+        %th Imię i Nazwisko
+        %th Email
+        %th Korespondujący?
+        %th Liczba porządkowa
+        %th
+    %tbody
+      - submission.authorships.each do |authorship|
+        - unless authorship.new_record?
+          %tr
+            %td= authorship.person.full_name
+            %td= authorship.person.email
+            %td= authorship.corresponding? ? raw("&#10004;") : ""
+            %td= authorship.position
+
+  = render partial: "author_form", locals: { authorship: authorship,
+                                             submission: submission }
+  
+  = button_to "Anuluj", controller: :public_submissions, action: :cancel, public_submission_id: submission.id
```

Comment:
Przydałoby się pytanie chroniące przed przypadkowym kliknięciem. Najlepiej byłoby to umieścić gdzieś daleko od guzika zatwierdzającego.